javascript-data-model/order-history.js

```diff
@@ -0,0 +1,99 @@
+var orderHistory = [
+  {
+    orderInformation: [
+      {
+        orderPlaced: 'August 4 2020',
+        total: '$34.00',
+        shipTo: 'JS Masher',
+        orderNumber: '114-3941689-87772352',
+        deliveredDate: 'August 8 2020'
+      }
+    ],
+    itemInformation: [
+      {
+        itemTitle: 'JavaScript for impatient programmers',
+        author: 'Rauschmayer, Dr. Axel',
+        returnWindow: 'Return window closed on Sep 7 2020',
+        price: '$31.55',
+        buyItAgain: null,
+        viewYourItem: null,
+        writeProductReview: null
+      }
+    ]
+  },
+  {
+    orderInformation: [
+      {
+        orderPlaced: 'July 19 2020',
+        total: '$44.53',
+        shipTo: 'JS Masher',
+        orderNumber: '113-9984268-1280257',
+        deliveredDate: 'July 20 2020'
+      }
+    ],
+    itemInformation: [
+      {
+        itemTitle: 'The Timeless Way of Building',
+        author: 'Alexander Christopher',
+        returnWindow: 'Return window closed on Aug 19 2020',
+        price: '$41.33',
+        buyItAgain: null,
+        viewYourItem: null,
+        writeProductReview: null
+      }
+    ]
+  },
+  {
+    orderInformation: [
+      {
+        orderPlaced: 'July 4 2020',
+        total: '$17.22',
+        shipTo: 'JS Masher',
+        orderNumber: '114-2875557-9059409',
+        deliveredDate: 'July 7 2020'
+      }
+    ],
+    itemInformation: [
+      {
+        itemTitle: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for Wii U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+        author: null,
+        returnWindow: 'Return window closed on Aug 5 2020',
+        price: '$15.98',
+        buyItAgain: null,
+        viewYourItem: null,
+        writeProductReview: null
+      }
+    ]
+  },
+  {
+    orderInformation: [
+      {
+        orderPlaced: 'July 3 2020',
+        total: '$138.93',
+        shipTo: 'JS Masher',
+        orderNumber: '113-2883177-2648248',
+        deliveredDate: 'July 5 2020'
+      }
+    ],
```

Comment:
Since order information never seems to have more than one object in it, we can just make it an object instead of an array.